### PR TITLE
Writing review of threat-modeling.en.md

### PR DESCRIPTION
**Pull request by KaiTebay**
Edit: Please ignore my messy commit history (unless it unknowingly affects my pull request)!

**Summary**

6 "et cetera" --> "etc.", as per [APA style](https://apastyle.apa.org/style-grammar-guidelines/abbreviations/latin).
6 Missing comma: subordinate clause.
8 Missing comma: embedded clause.
10 Missing comma: subordinate clause.
12 "security/privacy" --> "security and privacy", as per [APA Style](https://apastyle.apa.org/learn/faqs/use-slash-mark).
12 "potential" is redundant here: "could" expresses modality.
14 Simplified sentence structure: conciseness.
16 Updated to title case.
18 "want to" --> "should", as per [style guide](https://github.com/privacyguides/privacyguides.org/wiki/Writing-Style#use-must-for-requirements).
18 Are "those who might want to get ahold of your assets" not covered by the stressed definition of "adversary"? I couldn't think of a counter-example, so I omitted this for conciseness.
46 Removed "a": conciseness.
46 Missing commas: embedded clause.
46 Removed "at any likelihood": redundancy (already implied).
56 Added "of": accuracy.
56 Missing comma: embedded clause.
68 Partially updated to title case: "Protecting your belongings" reads like a title, whereas "Try it yourself:" is consistent with the tone of the other H3 headings (which are in sentence case).
72 Missing comma: subordinate clause.
82 "roommates/guests" --> "roommates or guests", as per [APA Style](https://apastyle.apa.org/learn/faqs/use-slash-mark).
86 "these" --> "those": accuracy.

- [x] Please check this box to confirm you have disclosed any relevant conflicts of interest in your post.
- [x] Please check this box to confirm your agreement to publish your work under the [Creative Commons Attribution-NoDerivatives 4.0 International](https://github.com/privacyguides/privacyguides.org/blob/main/LICENSE) license, and to grant Privacy Guides a perpetual, worldwide, non-exclusive, transferable, royalty-free, irrevocable license with the right to sublicense such rights through multiple tiers of sublicensees, to reproduce, modify, display, perform and distribute your contribution as part of our project.

### CODE REVIEW

**Review by mfwmyfacewhen — APPROVED** [review]

File: docs/basics/threat-modeling.en.md [hunk 1]
```diff
@@ -3,19 +3,19 @@ title: "Threat Modeling"
...
+Balancing security, privacy, and usability is one of the first and most difficult tasks you'll face on your privacy journey. Everything is a trade-off: The more secure something is, the more restricting or inconvenient it generally is, etc. Often, people find that the problem with the tools they see recommended is that they're just too hard to start using!
 
-If you wanted to use the **most** secure tools available, you'd have to sacrifice *a lot* of usability. And even then, ==nothing is ever fully secure.== There's **high** security, but never **full** security. That's why threat models are important.
+If you wanted to use the **most** secure tools available, you'd have to sacrifice *a lot* of usability. And, even then, ==nothing is ever fully secure.== There's **high** security, but never **full** security. That's why threat models are important.
 
-**So, what are these threat models anyways?**
+**So, what are these threat models, anyways?**
```
[the remark this answers is not in the document]
Comment (comastilt4901) [reply docs/basics/threat-modeling.en.md:10]: > This is more of a question than a correction. Is "anyways" or "anyway" correct here? @KaiTebay

From what I've searched, "anyways" is slang for/a colloquial variant/an informal form of "anyway" (amongst other things) and shouldn't be used for formal writing.

[the remark this answers is not in the document]
Comment (dngray) [reply docs/basics/threat-modeling.en.md:10]: > From what I've searched, "anyways" is slang for/a colloquial variant/an informal form of "anyway" (amongst other things) and shouldn't be used for formal writing.

100%. Not sure how this got in there. I feel this should be "anyway".

**Comment by dngray** [comment]
> Edit: Please ignore my messy commit history (unless it unknowingly affects my pull request)!

Doesn't matter, we can squash them into one before merging, so it's all good.

**Review by blacklight447 — APPROVED** [review]
Looks good.

**Review by dngray — APPROVED** [review]

**Pull request merged**